deploy/examples/monitoring/prometheusrule-default-values.yaml

```diff
@@ -0,0 +1,117 @@
+CephMgrIsAbsent:
```

Comment:
All of them are the same as they were in the hardcoded rule.
Except for the description of [CephOSDFlapping](https://github.com/rook/rook/blob/master/deploy/examples/monitoring/prometheus-ceph-v14-rules.yaml#L136), 

It was supposed to be "...has restarted **10** times in last 5 minutes..." instead of "...has restarted **5** times in last 5 minutes..." (As can be seen from the [expression](https://github.com/rook/rook/blob/master/deploy/examples/monitoring/prometheus-ceph-v14-rules.yaml#L141))

So, I fixed this also in this pr.